src/invoices/InvoicesList/InvoicesList.js

```diff
@@ -131,6 +151,18 @@ class InvoicesList extends Component {
           disableRecordCreation={disableRecordCreation}
           browseOnly={browseOnly}
           showSingleResult={showSingleResult}
+          editRecordComponent={InvoiceEdit}
+          newRecordInitialValues={{
```

Comment:
do you need this stub?

Reply:
at this stage - yes. It enables to create invoices